tests/pinrow.test.ts

```diff
@@ -5,5 +5,14 @@ import { fp } from "../src/footprinter"
 test("pinrow5", () => {
   const soup = fp.string("pinrow5").circuitJson()
   const svgContent = convertCircuitJsonToPcbSvg(soup)
+  const pinrowJson = fp.string("pinrow5_female").json()
+  expect(pinrowJson).toMatchObject({
+    fn: "pinrow",
+    num_pins: 5,
+    p: 2.54,
+    id: 1,
+    od: 1.5,
+    female: true,
```

Comment:
this would fail, a bit sus you didn't put this in actually 😉 

```suggestion
    male: false
```